custom/covid/tasks.py

```diff
@@ -17,11 +17,18 @@
 from dimagi.utils.logging import notify_exception
 from django.conf import settings
 
+# Include users that have synced in the last 48 hours
 SYNC_WINDOW_HOURS = 48
+
+# Exclude users that have synced in the last 8 hours
 SYNC_CUTOFF_HOURS = 8
 
+# Exclude users whose case load is less than this
 MIN_CASE_COUNT = 20000
 
+# Don't allow tasks to run beyond 11am UTC
```

Comment:
Why 11 AM UTC? Seems like the hours when it is allowed to run should possibly be configurable per domain?

Reply:
This is specific to USH domains. 11am UTC is when users start to come online.

Reply:
:+1: Sorry, wasn't looking at the file path.